cmd/manager/main.go

```diff
@@ -146,12 +152,24 @@ func main() {
 	}
 
 	cfg := config.NewDefaultConfiguration(mgr.GetClient(), &config.ConfigReader{})
+	setupLog.Info("argora", "version", bininfo.Version())
 
-	if err = controller.NewMetal3Reconciler(mgr.GetClient(), mgr.GetScheme(), cfg, netbox.NewNetbox(), flagVar.reconcileInterval).SetupWithManager(mgr, rateLimiter); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "metal3")
+	capiCRDExists, err := capiCRDExists(context.Background(), mgr.GetClient())
```

Comment:
Would it make sense to use a context with timeout, in case the apiserver isn't reachable?